pyproject.toml

```diff
@@ -19,7 +19,7 @@ py-buzz = "^3.1.0"
 # These must be included as a main dependency for the pytest extension to work out of the box
 respx = "^0.17.1"
 pytest = "^6.2.5"
-auto-name-enum = "^1.0.2"
+auto-name-enum = "^2"
```

Comment:
why such a change is needed?

Reply:
I changed `auto-name-enum` so that the default behavior preserve the casing of the values. So `SomeEnum.ALPHA` resolves to `"ALPHA"` instead of `"alpha"`. Prior to this, you would also have to inherit from the `NoMangleMixin` to have this behavior.

The reason for the casing to be changed for this project was due to its use in a few projects at my last job where the mapping to lower case was needed. Thus, I bumped auto-name-enum to `2.0.0`, since this was API breaking behavior.

I bumped it here because I wanted to use the newer version while I was at it.